enum/tarifstufe/tarifstufe.go

```diff
@@ -0,0 +1,20 @@
+package tarifstufe
+
+// Tarifstufe entspricht dem angegebenen Tarif aus der OBIS-Kennzahl mit der beanstandeten Energiemenge
+//
+//go:generate stringer --type Tarifstufe
+//go:generate jsonenums --type Tarifstufe
+type Tarifstufe int
+
+const (
+	TARIFSTUFE_0 Tarifstufe = iota + 20 // Z20
```

Comment:
why +20? is it to be consistent with Z20, Z21...?

Reply:
I wanted to do `iota`, so it'd start with 0, but then it would have a default value `TARIFSTUFE_0`, even when the value is not set. That would break things. So I opted to go for `iota + 20` to be consistent with the Z20, Z21... as the next best thing.